### ISSUE TITLE
pin - error nothing to repeat

### ISSUE THREAD

**Issue by Seraf** [mtpereira/ansible-erlang#1]
Hello,

First, thanks for your role !
I'm using it but have an error : 

```
TASK: [ansible-erlang | pin - set highest priority for this repository] ******* 
fatal: [10.1.0.86] => {'msg': 'error: nothing to repeat', 'failed': True}
fatal: [10.1.0.86] => {'msg': 'error: nothing to repeat', 'failed': True}
```

Do you have some info about it please ?

I'm using python 2.7.3 on Debian Wheezy with Ansible 1.9.2 and boto 2.38.0

Thanks

**Comment by mtpereira** [comment]
According to the error message, this is some issue while replacing the matched string on https://github.com/mtpereira/ansible-erlang/blob/master/templates/pin.j2#L2 .

I'm using Python 2.7.10 and I don't see this issue.  As far as I can tell, this is some issue with the Python's regex engine, since the same string is matching differently on different Python versions.

I'll try to workaround it by having a separate variable for the pin's origin, while trying to keep the same default behaviour.

**Comment by mtpereira** [comment]
Fixed on #2 .
